style.css

```diff
@@ -1,483 +1,744 @@
-  line-height: 24px;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  padding-top: 0.75rem;
+}
+
+.resumeLinkContainer {
+  padding-top: 2.25rem;
 }
 
-.about-me-section .resume {
-  padding: 12px;
-  background-color: #ff6b00;
+.resumeLink {
+  width: 9.75rem;
+  background-color: rgba(245, 88, 0, 1);
   border: none;
-  color: white;
-  width: 161px;
-  height: 48px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
+  color: #f4f1f0;
+  padding: 0.8rem;
+  text-align: center;
+  display: inline-block;
   font-weight: 700;
-  font-size: 15px;
+  font-size: 1rem;
+  cursor: pointer;
 }
 
-.about-me-section .resume:hover {
-  color: #f55800;
-  background-color: #fff5e1;
-  border: 1px solid #f55800;
+/* link button styles */
+.resumeLink:focus {
+  background-color: #f55800;
 }
 
-.about-me-section .resume:active {
-  color: white;
-  background-color: #e05304;
+.resumeLink:hover {
+  background-color: rgba(255, 245, 225, 1);
+  border: 2px solid rgba(245, 88, 0, 1);
+  color: rgba(245, 88, 0, 1);
 }
 
-.about-me-section .resume:disabled {
+.resumeLink:disabled {
   background-color: #f4f1f0;
+  border: none;
   color: #5b5957;
 }
 
-.about-me-section .skill-set ul {
-  padding: 0;
-  display: flex;
-  list-style: none;
-  text-align: center;
-}
-
-.about-me-section .skill-set .lang {
-  list-style-image: url("img/Rectangle 55.png");
+.background-img2 {
+  background-image: url('images/aboutMe backgroundMob.png');
+  background-size: contain;
+  width: 10rem;
+  height: 16.25rem;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 23.1rem;
+  right: 0;
 }
 
-.about-me-section .skill-set .fram {
-  list-style-image: url("img/Rectangle 56.png");
+.line2 {
+  padding-top: 12.5rem;
 }
 
-.about-me-section .skill-set .skills {
-  list-style-image: url("img/Rectangle 57.png");
+.experience {
+  color: #fff;
+  text-align: center;
+  padding-top: 5rem;
 }
 
-.about-me-section .skill-set .category {
-  font-family: 'InaiMathi', sans-serif;
+.experience h3 {
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
+  font-size: 2rem;
+  line-height: 2.75rem;
 }
 
-.about-me-section .skill-set .sublist {
-  list-style: none;
-  display: flex;
-  padding: 0;
-  font-family: 'Inter', sans-serif;
+.experience ul {
+  font-family: 'InaiMathi', 'Gill Sans', sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 22px;
-  line-height: 43px;
-  margin: 25px;
+  font-size: 1.3rem;
+  line-height: 2.6rem;
+  padding-top: 1.5rem;
+  padding-bottom: 1.5rem;
 }
 
-/* -------About me-------end */
-
-/* ---Contact section--- */
-
-.contact-section {
-  display: flex;
-  height: calc(100vh + 120px);
-  margin: 0;
-  margin-top: 100px;
-  padding: 0 24px;
-  flex-direction: column;
+/* contact section */
+.contact {
+  background-color: #fff;
+  display: grid;
+  padding: 4.3rem 0 9.6rem;
+  grid-template-columns: 20.5rem;
+  justify-content: center;
```

Comment:
- [ ] Great job implementing changes required by the previous reviewer. However, there're few things you need to work on before this milestone could meet its requirements. It is required to stick to the figma design as close as possible. This includes, font, typography, margin, padding among others. You should try as much as you can to produce a close replica of the figma design. Look at the red sqaure I used to box the two parts on the section. the width that your contact information covers must be the same with the figma file, likewise the form. There should not be too much obvious difference between your design and figma design. Kindly give this another shot. I believe you can do it. Let's try it one more time.

### Your Design
![your-form](https://user-images.githubusercontent.com/26597432/230625196-fa8f62f7-08d6-4918-b663-2c667a7d158a.png)

### Figma Design
![temp-form](https://user-images.githubusercontent.com/26597432/230625221-80e82aef-c783-468c-88f4-ec0c38cc72c2.png)